editor/src/components/window/file.rs

```diff
@@ -182,22 +182,10 @@ impl FileImplementedEditor for super::EchidnaWindow {
     }
```

Comment:
Can you please do this as well?

```
warning: unused import: `fmt`
 --> editor/src/components/window/file.rs:7:25
  |
7 | use std::{error::Error, fmt};
  |                         ^^^
  |
  = note: `#[warn(unused_imports)]` on by default

```

Reply:
done https://github.com/null8626/Echidna/commit/0e159c79bae10e868219cd3bff77355589bfb9ad